packages/js/migrations.json

```diff
@@ -47,6 +47,12 @@
       "version": "16.6.0-beta.0",
       "description": "Explicitly set 'updateBuildableProjectDepsInPackageJson' to 'true' in targets that rely on that value as the default.",
       "factory": "./src/migrations/update-16-6-0/explicitly-set-projects-to-update-buildable-deps"
+    },
+    "16-8-2-update-swcrc": {
```

Comment:
We also need a migration for package.json update right?

And we also update the version on `js` `versions.ts`

Reply:
Sorta, the version range already installed the latest, which is where the error came from. But we can update it more explicitly.